app/helpers/embed_player_helper.rb

```diff
@@ -37,59 +38,65 @@ def embed_player_episode_url(ep, type = nil, preview = false)
       params[EMBED_PLAYER_GUID] = ep.item_guid
     end
 
-    if type == "card" || type == "fixed_card"
+    if options.present? && options[:embed_player_type] == "card"
       params[EMBED_PLAYER_CARD] = "1"
     end
 
-    embed_params(params)
+    embed_params(params, preview)
   end
 
   def embed_player_podcast_url(podcast, options, preview = false)
     params = {}
     params[EMBED_PLAYER_FEED] = podcast.published_url
 
-    params[EMBED_PLAYER_PLAYLIST] = (options[:all_episodes] == "all") ? options[:all_episodes] : options[:episode_number]
-    params[EMBED_PLAYER_SEASON] = options[:season]
-    params[EMBED_PLAYER_CATEGORY] = options[:category]
+    if options[:all_episodes] === "all"
+      params[EMBED_PLAYER_PLAYLIST] = "all"
+    end
 
-    # TODO: the height styling doesn't really work here in the way Play specifies how it needs to be for a playlist, so leaving this as a TODO for now.
-    # if options[:embed_player_type] == "card" || options[:embed_player_type] == "fixed_card"
-    #   params[EMBED_PLAYER_CARD] = "1"
-    # end
+    if options[:all_episodes] == "number" && options[:episode_number].to_i > 1
```

Comment:
`> 1` not `>=` 1?

Reply:
1 episode a playlist doesn't make. Playlist of 1 episode is just the player.

Reply:
Oh, I see.  That makes sense.